forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -2412,6 +2431,11 @@ public void dealDamage(final boolean isCombat, final CardDamageMap damageMap, fi
                         // CR 120.10
                         lethal = c.isCreature() ? Math.min(lethal, lethalPW) : lethalPW;
                     }
+                    if (c.isBattle()) {
+                        int lethalBattle = c.getCurrentDefense();
+
+                        lethal = c.isCreature() || c.isPlaneswalker() ? Math.min(lethal, lethalBattle) : lethalBattle;
+                    }
```

Comment:
This "if" should also be reproduced before this line:
https://github.com/Card-Forge/forge/blob/5dc5bc3bfb9ee0ec11d54df1d139f0ec90174985/forge-game/src/main/java/forge/game/ability/effects/DamageDealEffect.java#L303

Reply:
Thats dumb. We should just pull out to live on Card "calculateLethalDamage"